src/main/java/com/plusls/ommc/mixin/feature/forceBreakingCooldown/MixinClientPlayerInteractionManager.java

```diff
@@ -16,9 +16,7 @@
     private int blockBreakingCooldown;
 
     @Inject(method = "attackBlock",
-            at = @At(value = "INVOKE",
-                    target = "Lnet/minecraft/client/network/ClientPlayerInteractionManager;breakBlock(Lnet/minecraft/util/math/BlockPos;)Z",
-                    ordinal = 1))
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/client/network/ClientPlayerInteractionManager;breakBlock(Lnet/minecraft/util/math/BlockPos;)Z"))
```

Comment:
I think it need `ordinal`

Reply:
After adding this, the game can't launch with optifine. It says `Field not found: World.tileEntitiesToBeRemoved`. But it can run without optifine.  I have no idea about this.

Reply:
Because `ordinal` starts from `0`, and there's only one `breakBlock`, if you use `ordinal = 1`, strange things like this may happen. If you persist in adding this, my suggestion is to add `ordinal = 0`.

Reply:
Yes, I means add `ordinal = 0`